lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -1332,12 +1375,16 @@ struct FIRRTLLowering : public FIRRTLVisitor<FIRRTLLowering, LogicalResult> {
   /// This is true if we've emitted `INIT_RANDOM_PROLOG_ into an initial
   /// block in this module already.
   bool randomizePrologEmitted;
+  /// This is true if this module is part of the design under test hierarchy.
+  bool isDUT;
```

Comment:
Mega-nit: this is really "is DUT or is inside DUT". A more descriptive name here might makes sense. It may be clearer to call this `inDUTHierarchy` or something like that. It is also not strictly necessary that the DUT is included in this. I.e., by refactoring the code to only include modules under the DUT, this would then be `insideDUT` or something like that.